src/users/usersReducers.js

```diff
@@ -13,10 +13,15 @@ import { NULL_ARRAY } from '../nullObjects';
 
 const initialState: UsersState = NULL_ARRAY;
 
-const realmInit = (state: UsersState, action: RealmInitAction): UsersState => {
-  console.log('action.data', action.data);
-  return action.data.realm_users;
-};
+const realmInit = (state: UsersState, action: RealmInitAction): UsersState => [
+  // Object.assign(x, { is_nonactive: false }) modifies the data in `x` and thus in `action`!
+  // Object.assign({ is_nonactive: true }, x) wouldn't, but { is_nonactive: true } is a sealed
+  // object, so no properties can be added. {} is unsealed, so Object.assign({}, { is_nonactive: false }, x))
+  // works. Credits to http://also.github.io/notes/2016/03/13/flow-object-assign.html
+  ...action.data.realm_users.map(x => Object.assign({}, { is_nonactive: false }, x)),
```

Comment:
While I don't agree with the direction of the commit, here is a tip for the code in general ;)

```js
*.map(x => Object.assign({}, { is_nonactive: false }, x))
```
can become:
```js
*.map(x => {
  ...x,
  is_nonactive: false,
})
```

Reply:
Thanks, that looks better!